services/api/src/typeDefs.js

```diff
@@ -278,17 +278,21 @@ const typeDefs = gql`
     uptimeRobotStatusPageId: String
   }
 
-  type Openshift {
+  type Kubernetes {
```

Comment:
I thought we'd decided to abstract the type naming to a `DeployTarget` or similar instead of Openshift or Kubernetes, and then specify the type of deploytarget - be it Openshift, Kubernetes or whatever comes next

Reply:
Yes, based on the conversations here: https://github.com/amazeeio/lagoon/pull/1582 I think you are right. I'll touch base with Brandon on this point tomorrow morning.

Reply:
I would still keep the object `Kubernetes`, but on the project itself we use as the key `deployTarget` to reference to which kubernetes we want to deploy too. With that we could in the future also add a new object for another type of cluster, which then also can be referenced under `deployTarget` in the the project.

Reply:
I thought we decided kubernetes was "the one" so didn't abstract it at all. In order to avoid future breaking changes in case something new comes up, I did add in an interface, but still keeping the single `Kubernetes` object with a field that defines whether it's openshift/k8s under the hood.